config/values.yml

```diff
@@ -156,3 +156,6 @@ metric_proxy:
     secret_name: "metric-proxy-cert"
     crt: ""
     key: ""
+
+opi:
+  unsafe_allow_automount_service_account_token: false
```

Comment:
Is this configuration used anywhere? I didn't see a reference to `data.values.opi.unsafe_allow_automount_service_account_token`.

Reply:
You're right, this isn't used anywhere. This was from a different approach we thought about going down when we first started this work.

We'll remove this.

Reply:
Removed